### ISSUE TITLE
Unexpected className found parsing IDEActivityLogMessage IDEActivityLogActionMessage

### ISSUE THREAD

**Issue by mollyIV** [MobileNativeFoundation/XCLogParser#187]
It seems that with the introduction of **Xcode 15**, there's a new "class instance" type: `IDEActivityLogActionMessage`. 

An example:

```
1(27%IDEActivityLogActionMessage6@78"The identity of “XYZ.xcframework” is not recorded in your project.-123213#432423423423423#0#-0#-23%DVTTextDocumentLocation7@93"file:///Users/blob/Projects/foo/Carthage/Build/XYZ.xcframework/0000000000000000^0#0#0#0#13333073712324212#0#0#---110"{"frameworkSigningMismatch":{"_0":{"frameworkPath":"\/Users\/blob\/Projects\/foo\/Carthage\/Build\/XYZ.xcframework"}}}0#0#0#--3333"ProcessXCFramework /Users/blob/Projects/foo/Carthage/Build/XYZ.xcframework /Users/blob/Library/Developer/Xcode/DerivedData/foo-abc/Build/Products/foo-debug-iphoneos/XYZ.framework ios
    cd /Users/blob/Projects/foo/Projects/foo
    builtin-process-xcframework --xcframework /Users/blob/Projects/foo/Carthage/Build/XYZ.xcframework --platform ios --target-path /Users/blob/Library/Developer/Xcode/DerivedData/foo-abc/Build/Products/foo-debug-iphoneos

    Task input dependencies:
        /Users/blob/Projects/foo/Carthage/Build/XYZ.xcframework/
        /Users/blob/Library/Developer/Xcode/DerivedData/foo-abc/Build/Products/foo-debug-iphoneos

    Task output dependencies:
        /Users/blob/Library/Developer/Xcode/DerivedData/foo-abc/Build/Products/foo-debug-iphoneos/XYZ.framework
        /Users/blob/Library/Developer/Xcode/DerivedData/foo-abc/Build/Products/foo-debug-iphoneos/XYZ.framework/Headers
        /Users/blob/Library/Developer/Xcode/DerivedData/foo-abc/Build/Products/foo-debug-iphoneos/XYZ.framework/Headers/XYZ.h
        /Users/blob/Library/Developer/Xcode/DerivedData/foo-abc/Build/Products/foo-debug-iphoneos/XYZ.framework/XYZ
        /Users/blob/Library/Developer/Xcode/DerivedData/foo-abc/Build/Products/foo-debug-iphoneos/XYZ.framework/Info.plist
        /Users/blob/Library/Developer/Xcode/DerivedData/foo-abc/Build/Products/foo-debug-iphoneos/XYZ.framework/Modules
        /Users/blob/Library/Developer/Xcode/DerivedData/foo-abc/Build/Products/foo-debug-iphoneos/XYZ.framework/Modules/module.modulemap
```

According to the [XCActivitylog Format](https://github.com/MobileNativeFoundation/XCLogParser/blob/dfce1e401fe7ba45217a56d69eb48403897a506d/docs/Xcactivitylog%20Format.md?plain=1)  documentation, we need to obtain the properties of this class from a private `IDEFoundation.framework` framework. Unfortunately, I wasn't able to do so. 

If anyone could help to get the list of properties we should include in a new `IDEActivityLogActionMessage.swift`, I'd be happy to implement it 🙏

**Comment by BalestraPatrick** [comment]
Hello @mollyIV, I tried using class-dump as mentioned in the documentation, but it's possible that the code in `IDEFoundation.framework` has been rewritten into Swift as it's not really providing any useful output to me. By using Hopper, I found this.
<img width="694" alt="Screenshot 2023-10-13 at 10 21 56 PM" src="https://github.com/MobileNativeFoundation/XCLogParser/assets/3658887/c26d9d6e-02a9-4a48-bbe0-8b391bfc3813">

I'm not exactly sure what "action" refers to (maybe just a string?), but I believe the other 3 fields are similar to others that we have in other model objects. It's worth trying to come up with a model and see if parsing your xcactivitylog parses it correctly.
